apps/vaporgui/TFMapWidget.cpp

```diff
@@ -199,6 +201,11 @@ TFMapWidget::TFMapWidget(TFMap *map)
     AddMap(map);
     setContextMenuPolicy(Qt::CustomContextMenu);
     connect(this, SIGNAL(customContextMenuRequested(const QPoint &)), this, SLOT(_showContextMenu(const QPoint &)));
+
+    // Is there a better way or place to set the sizePolicy of this object?
```

Comment:
You could do it in the subclass constructor

Reply:
The subclass isn't a QWidget though.  It needs to be done in the TFMapWidget, not the TFMap :\

Reply:
TFMap could have a virtual `GetSetSizePolicy` method.

Reply:
Ok, that's a good idea.